chatkit-core/src/main/kotlin/com/pusher/chatkit/SynchronousCurrentUser.kt

```diff
@@ -28,7 +28,7 @@ class SynchronousCurrentUser(
         private val pushNotifications: PushNotifications?
 ) {
 
-    val rooms: List<Room> get() = chatManager.roomService.roomStore.toList()
+    val rooms: List<Room> get() = chatManager.roomService.roomStore.listAll()
```

Comment:
What's the motivation for this change? Kotlin convention is to call it `toList`: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/to-list.html

Reply:
I don't remember any more. I'll revert.